sdccc/src/main/java/com/draeger/medical/sdccc/tests/biceps/invariant/InvariantParticipantModelAnnexTest.java

```diff
@@ -166,6 +169,9 @@ void testRequirementB284() throws NoTestData, IOException {
         assertTestData(acceptableSequenceSeen.get(), "No acceptable sequence seen, test failed.");
     }
 
+    // NOTE: No PreCondition can ensure that an ApprovedJurisdictions list is present or absent as this
+    //       list is supposed to be static for a Vmd. When all Vmds in a Device's Mdib have an ApprovedJurisdictions
+    //       list, then this test case should not be applied to it.
```

Comment:
This is definitely not the right place to "inform" anyone when not to apply the test, no user will ever read it.

Reply:
Made sure this is also stated in RQM.